### PR TITLE
Allow users to set TEST_TIME and TRSS_URL in testJobTemplate

### PR DESCRIPTION
**Pull request by llxia**
Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

**Review by renfeiw — APPROVED** [review]
LGTM

File: buildenv/jenkins/testJobTemplate [hunk 1]
```diff
@@ -348,7 +350,12 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							}
 							choiceParam('PARALLEL', PARALLEL_LIST, "Optional. Parallel mode")
 							stringParam('NUM_MACHINES', NUM_MACHINES, "Optional. Number of machines to run in parallel. Need to be used with PARALLEL=Dynamic")
-							booleanParam('PERSONAL_BUILD', false, "Is this a personal build?")
+							stringParam('TEST_TIME', TEST_TIME, '''Optional. Expected overall test build execution time. Need to be used with PARALLEL=Dynamic.<br>
```
Comment (smlambert) [inline buildenv/jenkins/testJobTemplate:353]: Minor: Need**s** to be used versus Need to be used on line 353 and 357

**Review by smlambert — APPROVED** [review]
Minor grammar if you feel like correcting it before we merge this.

File: buildenv/jenkins/testJobTemplate (hunk 1 above)
Comment (llxia) [reply buildenv/jenkins/testJobTemplate]: updated. Thanks

**Pull request merged**